pkg/dbmate/migration.go

```diff
@@ -66,6 +67,7 @@ var (
 	whitespaceRegExp      = regexp.MustCompile(`\s+`)
 	optionSeparatorRegExp = regexp.MustCompile(`:`)
 	blockDirectiveRegExp  = regexp.MustCompile(`^--\s*migrate:(up|down)`)
+	separatorRegExp       = regexp.MustCompile(`(?m)^\s*--\s*migrate:separator\s*$`)
```

Comment:
If we support multiple up- and down-migrations in a single file, we don't need a `-- migrate:statement-end` since each statement is already ended by either a subsequent `-- migrate:up` or `-- migrate:down` or EOF.

Reply:
Thanks for the tip, I've used `-- migrate:up` as a separator to split file by migration sections and process each section with existing logic